src/components/canvas/Background/index.ts

```diff
@@ -0,0 +1,77 @@
+import DisplayObject, { IProps } from 'components/canvas/DisplayObject';
+import Sprite, { ISpriteProps } from 'components/canvas/Sprite';
+import { GAME_BG_SPRITE_SIZE } from 'utils/constants';
+import SpriteSheet from '../../../assets/sprites/spriteSheet.png';
+
+export interface IBackGroundProps extends IProps {
+    width: number
+    height: number
+}
+
+class Background extends DisplayObject<IBackGroundProps> {
+    private xRowsNum: number;
+
+    private yRowsNum: number;
+
+    private levelMetaData: Array<Array<Sprite<ISpriteProps>>> = [];
```

Comment:
Такая запись читается проще, кмк

```suggestion
    private levelMetaData: Sprite<ISpriteProps>[][] = [];
```

Reply:
изначально тоже так написал - на это ругается линтер, правило `@typescript-eslint/array-type`